<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QAruCo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="cameraLayout"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListView" name="markersList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="currentConfiguration">
          <property name="text">
           <string>Текущий блок</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="configurationIdLabel">
          <property name="text">
           <string>ID блока</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="configurationIdValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="configurationDateLabel">
          <property name="text">
           <string>Дата создания</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="configurationDateValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="configurationTypeLabel">
          <property name="text">
           <string>Тип блока</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="configurationTypeValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="configurationNameLabel">
          <property name="text">
           <string>Название блока</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="configurationNameValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="cameraInfoAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="markersMenu">
    <property name="title">
     <string>Маркеры</string>
    </property>
    <addaction name="detectBlocksAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="markersMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="cameraInfoAction"/>
   <addaction name="separator"/>
  </widget>
  <action name="cameraInfoAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/info.png</normaloff>:/icons/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>Список камер</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="detectBlocksAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/detector.png</normaloff>:/icons/icons/detector.png</iconset>
   </property>
   <property name="text">
    <string>Поиск блоков</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
